backend/src/main/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+    <threadsafe>false</threadsafe>
+    <sessions-enabled>true</sessions-enabled>
+    <runtime>java8</runtime>
+    <static-files>
+        <!-- prevent unwanted caching when accessing via the web preview server -->
+        <include path="/**" expiration="0s" />
+    </static-files>
```

Comment:
That's because I changed the output to webapp instead of dist, to avoid the move command. So, every-time the project is build it will remove WEB-INF and gets another copy from an outer directory. I assumed that will save time as our project gets bigger.

Reply:
Yeah, this is fine. During the build phase, Maven builds our Angular app into the `webapp` directory (in doing so, it deletes everything in it). Then, we make a copy of the `WEB-INF` directory and its children in the `webapp` directory because Angular cannot build these files. It might make more sense to store the `WEB-INF` directory inside of another directory to keep all files that aren't built by Angular in one place.